src/bench/chacha20.cpp

```diff
@@ -41,6 +41,10 @@ static void CHACHA20_1MB(benchmark::State& state)
     CHACHA20(state, BUFFER_SIZE_LARGE);
 }
 
-BENCHMARK(CHACHA20_64BYTES, 500000);
-BENCHMARK(CHACHA20_256BYTES, 250000);
-BENCHMARK(CHACHA20_1MB, 340);
+//TODO add back below once benchmarking backports are done
+//BENCHMARK(CHACHA20_64BYTES, 500000);
+//BENCHMARK(CHACHA20_256BYTES, 250000);
+//BENCHMARK(CHACHA20_1MB, 340);
+BENCHMARK(CHACHA20_64BYTES);
+BENCHMARK(CHACHA20_256BYTES);
+BENCHMARK(CHACHA20_1MB);
```

Comment:
Don't let Nathan see this :P (missing new-line)

Reply:
:D I actually did notice this yesterday. It sticks out like a sore thumb. 👍

Reply:
resolved and then squashed